snmp/datadog_checks/snmp/data/default_profiles/anue-packet-broker.yaml

```diff
@@ -0,0 +1,10 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.32620.1.1
```

Comment:
Do you know why it's `1.3.6.1.4.1.32620.1.1` and not `1.3.6.1.4.1.32620.1.*` ?
looking at the [ANUE-MIB](http://www.circitor.fr/Mibs/Html/A/ANUE-MIB.php), the oid `1.3.6.1.4.1.32620.1.1` is for [anueNtoObjects](http://www.circitor.fr/Mibs/Html/A/ANUE-MIB.php#anueNtoObjects) and `1.3.6.1.4.1.32620.1.2` is for [anueGscObjects](http://www.circitor.fr/Mibs/Html/A/ANUE-MIB.php#anueGscObjects)

Reply:
This profile was supposed to target packet brokers specifically, but since it does not add specific metrics we can indeed make it generic to `anue`

Reply:
Done 🙂